main.cpp

```diff
@@ -52,106 +52,106 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: mana
+//  action 1: check amount of mana
+mana.checkAmount();
+//  action 2: use mana
+mana.useMana();
+//  action 3: refill mana
+mana.refillMana();
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: account
+//  action 1: check balance
+account.checkBalance();
+//  action 2: debit funds
+account.debitFunds();
+//  action 3: credit funds
+account.creditFunds();
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: distance
+//  action 1: total distance
+distance.totalDistanceTravelled();
+//  action 2: distance remaining
+distance.distanceLeftToGo();
+//  action 3: convert distance to km
+distance.convertToKm();
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: list
+//  action 1: add item
+list.addItem();
+//  action 2: remove item
+list.removeItem();
+//  action 3: increase item count
+list.multiplyItem();

```

Comment:
As explained in the instructions:
Here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.